src/screens/Processors/ProcessorCards.res

```diff
@@ -214,21 +242,31 @@ let make = (
     <div className="flex flex-col gap-4">
       {if showIcons {
         <>
-          {connectorListFiltered->iconsConnectors("Connect a new processor", true, ())}
-          {<RenderIf condition={featureFlagDetails.testProcessors && showTestProcessor}>
-            {featureFlagDetails.testProcessors
-            ->dummyConnectorList
-            ->iconsConnectors("Connect a test processor", false, ~showSearch=false, ())}
-          </RenderIf>}
+          {connectorListFiltered->iconsConnectors(
+            "Connect a new processor",
+            true,
+            ~showDummyConnectorButton=true,
+            (),
+          )}
+          // {<RenderIf condition={featureFlagDetails.testProcessors && showTestProcessor}>
+          //   {featureFlagDetails.testProcessors
+          //   ->dummyConnectorList
+          //   ->iconsConnectors("Connect a test processor", false, ~showSearch=false, ())}
+          // </RenderIf>}
         </>
       } else {
         <>
-          <RenderIf condition={featureFlagDetails.testProcessors && showTestProcessor}>
-            {featureFlagDetails.testProcessors
-            ->dummyConnectorList
-            ->descriptedConnectors("Connect a test processor", false, ~showSearch=false, ())}
-          </RenderIf>
-          {connectorListFiltered->descriptedConnectors("Connect a new processor", true, ())}
+          // <RenderIf condition={featureFlagDetails.testProcessors && showTestProcessor}>
+          //   {featureFlagDetails.testProcessors
+          //   ->dummyConnectorList
+          //   ->descriptedConnectors("Connect a test processor", false, ~showSearch=false, ())}
+          // </RenderIf>
+          {connectorListFiltered->descriptedConnectors(
```

Comment:
In descriptedConnectors try to give named Parameters so it would be better for understanding what is true and all.